docker/cygnus-ngsi/Dockerfile

```diff
@@ -254,7 +254,9 @@ ENV CYGNUS_ARCGIS_BATCH_TTL ""
 ENV MVN_VER "3.5.4"
 ENV MVN_HOME "/opt/apache-maven"
 
-ENV FLUME_VER "1.9.0"
+ENV FLUME_VER "1.11.0"
+ENV FLUME_TGZ "apache-flume-${FLUME_VER}-bin.tar.gz"
+ENV FLUME_URL "https://archive.apache.org/dist/flume/${FLUME_VER}/${FLUME_TGZ}"
```

Comment:
```suggestion
```

I guess this is not needed also (related with https://github.com/telefonicaid/fiware-cygnus/pull/2212#discussion_r1073187491)

Reply:
Maybe I'm wrong... I see later in the file:

```
    curl --remote-name --location --insecure --silent --show-error "${FLUME_URL}" && \
    tar zxf apache-flume-${FLUME_VER}-bin.tar.gz && \
    mv apache-flume-${FLUME_VER}-bin ${FLUME_HOME} && \
```

So I'm not sure how it works :)

Maybe we should discuss at skype.